src/ini-load.js

```diff
@@ -31,14 +32,43 @@ const EXCLUDE_INI_LOAD = [
  * @param {!Element|!./service/ampdoc-impl.AmpDoc} elementOrAmpDoc
  * @param {!Window} hostWin
  * @param {!./layout-rect.LayoutRectDef} rect
- * @param {boolean=} opt_isInPrerender signifies if we are in prerender mode.
+ * @param {boolean=} opt_prerenderableOnly signifies if we are in prerender mode.
  * @return {!Promise}
  */
 export function whenContentIniLoad(
   elementOrAmpDoc,
   hostWin,
   rect,
-  opt_isInPrerender
+  opt_prerenderableOnly
+) {
+  // eslint-disable-next-line no-undef
```

Comment:
Seems like this `eslint` directive is no longer necessary: https://app.circleci.com/pipelines/github/ampproject/amphtml/1495/workflows/2b3584d4-a77c-4bf6-a9d4-fa5e4ebae85e/jobs/14278/parallel-runs/0/steps/0-107

![image](https://user-images.githubusercontent.com/26553114/107323559-7795f900-6a74-11eb-9483-59335fb9d296.png)